python/src/DistributionImplementation_doc.i.in

```diff
@@ -504,13 +511,13 @@ OT_Distribution_computeLogGeneratingFunction_doc
 
 Parameters
 ----------
-X : sequence of float, 2-d sequence of float
-    PDF input(s).
+x : sequence of float, 2-d sequence of float
+    Point in :math:`\Rset^d` such that the PDF at this point is non equal to 0.
 
 Returns
 -------
 f : float, :class:`~openturns.Point`
-    Logarithm of the PDF value(s) at input(s) :math:`X`."
+    Logarithm of the PDF at `x`."
```

Comment:
You removed the `:math` formatting, which is good. Hence, the input argument is written `*x*`:
```suggestion
    Logarithm of the PDF at *x*."
```
I suggest to fix the other occurence of this pattern before and after this line.